{
  "cryptography":
  [
    {
      "description": "A flaw was found in the python-cryptography package. This issue may allow a remote attacker to decrypt captured messages in TLS servers that use RSA key exchanges, which may lead to exposure of confidential or sensitive data.",
      "vulnerability_id": "CVE-2023-50782",
      "name": "CVE-2023-50782",
      "package_name": "cryptography",
      "package_details":
      {
        "file_path": "opt/conda/lib/python3.9/site-packages/cryptography-41.0.7.dist-info/METADATA",
        "name": "cryptography",
        "package_manager": "PYTHONPKG",
        "version": "41.0.7",
        "release": null
      },
      "remediation":
      {
        "recommendation":
        {
          "text": "None Provided"
        }
      },
      "cvss_v3_score": 7.5,
      "cvss_v30_score": 0.0,
      "cvss_v31_score": 7.5,
      "cvss_v2_score": 0.0,
      "cvss_v3_severity": "HIGH",
      "source_url": "https://nvd.nist.gov/vuln/detail/CVE-2023-50782",
      "source": "NVD",
      "severity": "HIGH",
      "status": "ACTIVE",
      "title": "CVE-2023-50782 - cryptography",
      "reason_to_ignore": "N/A"
    }
  ],
  "transformers":
  [
    {
      "description": "Deserialization of Untrusted Data in GitHub repository huggingface/transformers prior to 4.36.",
      "vulnerability_id": "CVE-2023-6730",
      "name": "CVE-2023-6730",
      "package_name": "transformers",
      "package_details":
      {
        "file_path": "opt/conda/lib/python3.9/site-packages/transformers-4.26.0.dist-info/METADATA",
        "name": "transformers",
        "package_manager": "PYTHONPKG",
        "version": "4.26.0",
        "release": null
      },
      "remediation":
      {
        "recommendation":
        {
          "text": "None Provided"
        }
      },
      "cvss_v3_score": 8.8,
      "cvss_v30_score": 0.0,
      "cvss_v31_score": 8.8,
      "cvss_v2_score": 0.0,
      "cvss_v3_severity": "HIGH",
      "source_url": "https://nvd.nist.gov/vuln/detail/CVE-2023-6730",
      "source": "NVD",
      "severity": "HIGH",
      "status": "ACTIVE",
      "title": "CVE-2023-6730 - transformers",
      "reason_to_ignore": "N/A"
    },
    {
      "description": "Deserialization of Untrusted Data in GitHub repository huggingface/transformers prior to 4.36.",
      "vulnerability_id": "CVE-2023-7018",
      "name": "CVE-2023-7018",
      "package_name": "transformers",
      "package_details":
      {
        "file_path": "opt/conda/lib/python3.9/site-packages/transformers-4.26.0.dist-info/METADATA",
        "name": "transformers",
        "package_manager": "PYTHONPKG",
        "version": "4.26.0",
        "release": null
      },
      "remediation":
      {
        "recommendation":
        {
          "text": "None Provided"
        }
      },
      "cvss_v3_score": 7.8,
      "cvss_v30_score": 0.0,
      "cvss_v31_score": 7.8,
      "cvss_v2_score": 0.0,
      "cvss_v3_severity": "HIGH",
      "source_url": "https://nvd.nist.gov/vuln/detail/CVE-2023-7018",
      "source": "NVD",
      "severity": "HIGH",
      "status": "ACTIVE",
      "title": "CVE-2023-7018 - transformers",
      "reason_to_ignore": "N/A"
    }
  ],
  "libzmq5":
  [
    {
      "reason_to_ignore": "The current version is marked as a fix version in the CVE notes, and this is the latest version.",
      "description": "\n It was discovered that ZeroMQ incorrectly handled certain application metadata.\n A remote attacker could use this issue to cause ZeroMQ to crash, or possibly\n execute arbitrary code.",
      "vulnerability_id": "CVE-2019-13132",
      "name": "CVE-2019-13132",
      "package_name": "libzmq5",
      "package_details":
      {
        "file_path": null,
        "name": "libzmq5",
        "package_manager": "OS",
        "version": "4.3.2",
        "release": "2ubuntu1"
      },
      "remediation":
      {
        "recommendation":
        {
          "text": "None Provided"
        }
      },
      "cvss_v3_score": 9.8,
      "cvss_v30_score": 0,
      "cvss_v31_score": 9.8,
      "cvss_v2_score": 7.5,
      "cvss_v3_severity": "CRITICAL",
      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2019/CVE-2019-13132.html",
      "source": "UBUNTU_CVE",
      "severity": "HIGH",
      "status": "ACTIVE",
      "title": "CVE-2019-13132 - libzmq5"
    }
  ],
  "linux":[
      {
         "description":" In the Linux kernel, the following vulnerability has been resolved: firmware: arm_scmi: Harden accesses to the reset domains Accessing reset domains descriptors by the index upon the SCMI drivers requests through the SCMI reset operations interface can potentially lead to out-of-bound violations if the SCMI driver misbehave. Add an internal consistency check before any such domains descriptors accesses.",
         "vulnerability_id":"CVE-2022-48655",
         "name":"CVE-2022-48655",
         "package_name":"linux",
         "package_details":{
            "file_path":null,
            "name":"linux",
            "package_manager":"OS",
            "version":"5.4.0",
            "release":"189.209"
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.8,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.8,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-48655.html",
         "source":"UBUNTU_CVE",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2022-48655 - linux",
         "reason_to_ignore":"N/A"
      },
      {
         "description":" In the Linux kernel, the following vulnerability has been resolved: net: fix __dst_negative_advice() race __dst_negative_advice() does not enforce proper RCU rules when sk->dst_cache must be cleared, leading to possible UAF. RCU rules are that we must first clear sk->sk_dst_cache, then call dst_release(old_dst). Note that sk_dst_reset(sk) is implementing this protocol correctly, while __dst_negative_advice() uses the wrong order. Given that ip6_negative_advice() has special logic against RTF_CACHE, this means each of the three ->negative_advice() existing methods must perform the sk_dst_reset() themselves. Note the check against NULL dst is centralized in __dst_negative_advice(), there is no need to duplicate it in various callbacks. Many thanks to Clement Lecigne for tracking this issue. This old bug became visible after the blamed commit, using UDP sockets.",
         "vulnerability_id":"CVE-2024-36971",
         "name":"CVE-2024-36971",
         "package_name":"linux",
         "package_details":{
            "file_path":null,
            "name":"linux",
            "package_manager":"OS",
            "version":"5.4.0",
            "release":"189.209"
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.8,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.8,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://people.canonical.com/~ubuntu-security/cve/2024/CVE-2024-36971.html",
         "source":"UBUNTU_CVE",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2024-36971 - linux",
         "reason_to_ignore":"N/A"
      },
      {
         "description":" In the Linux kernel, the following vulnerability has been resolved: erofs: fix pcluster use-after-free on UP platforms During stress testing with CONFIG_SMP disabled, KASAN reports as below: ================================================================== BUG: KASAN: use-after-free in __mutex_lock+0xe5/0xc30 Read of size 8 at addr ffff8881094223f8 by task stress/7789 CPU: 0 PID: 7789 Comm: stress Not tainted 6.0.0-rc1-00002-g0d53d2e882f9 #3 Hardware name: Red Hat KVM, BIOS 0.5.1 01/01/2011 Call Trace: <TASK> .. __mutex_lock+0xe5/0xc30 .. z_erofs_do_read_page+0x8ce/0x1560 .. z_erofs_readahead+0x31c/0x580 .. Freed by task 7787 kasan_save_stack+0x1e/0x40 kasan_set_track+0x20/0x30 kasan_set_free_info+0x20/0x40 __kasan_slab_free+0x10c/0x190 kmem_cache_free+0xed/0x380 rcu_core+0x3d5/0xc90 __do_softirq+0x12d/0x389 Last potentially related work creation: kasan_save_stack+0x1e/0x40 __kasan_record_aux_stack+0x97/0xb0 call_rcu+0x3d/0x3f0 erofs_shrink_workstation+0x11f/0x210 erofs_shrink_scan+0xdc/0x170 shrink_slab.co",
         "vulnerability_id":"CVE-2022-48674",
         "name":"CVE-2022-48674",
         "package_name":"linux",
         "package_details":{
            "file_path":null,
            "name":"linux",
            "package_manager":"OS",
            "version":"5.4.0",
            "release":"189.209"
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.8,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.8,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-48674.html",
         "source":"UBUNTU_CVE",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2022-48674 - linux",
         "reason_to_ignore":"N/A"
      },
      {
         "description":" In the Linux kernel, the following vulnerability has been resolved: RDMA/mlx5: Fix fortify source warning while accessing Eth segment ------------[ cut here ]------------ memcpy: detected field-spanning write (size 56) of single field \"eseg->inline_hdr.start\" at /var/lib/dkms/mlnx-ofed-kernel/5.8/build/drivers/infiniband/hw/mlx5/wr.c:131 (size 2) WARNING: CPU: 0 PID: 293779 at /var/lib/dkms/mlnx-ofed-kernel/5.8/build/drivers/infiniband/hw/mlx5/wr.c:131 mlx5_ib_post_send+0x191b/0x1a60 [mlx5_ib] Modules linked in: 8021q garp mrp stp llc rdma_ucm(OE) rdma_cm(OE) iw_cm(OE) ib_ipoib(OE) ib_cm(OE) ib_umad(OE) mlx5_ib(OE) ib_uverbs(OE) ib_core(OE) mlx5_core(OE) pci_hyperv_intf mlxdevm(OE) mlx_compat(OE) tls mlxfw(OE) psample nft_fib_inet nft_fib_ipv4 nft_fib_ipv6 nft_fib nft_reject_inet nf_reject_ipv4 nf_reject_ipv6 nft_reject nft_ct nft_chain_nat nf_nat nf_conntrack nf_defrag_ipv6 nf_defrag_ipv4 ip_set nf_tables libcrc32c nfnetlink mst_pciconf(OE) knem(OE) vfio_pci vfio_pci_core vfio_iommu_type1 vfio iommufd irqby",
         "vulnerability_id":"CVE-2024-26907",
         "name":"CVE-2024-26907",
         "package_name":"linux",
         "package_details":{
            "file_path":null,
            "name":"linux",
            "package_manager":"OS",
            "version":"5.4.0",
            "release":"189.209"
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.8,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.8,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://people.canonical.com/~ubuntu-security/cve/2024/CVE-2024-26907.html",
         "source":"UBUNTU_CVE",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2024-26907 - linux",
         "reason_to_ignore":"N/A"
      },
      {
         "description":" In the Linux kernel, the following vulnerability has been resolved: smb: client: fix potential OOBs in smb2_parse_contexts() Validate offsets and lengths before dereferencing create contexts in smb2_parse_contexts(). This fixes following oops when accessing invalid create contexts from server: BUG: unable to handle page fault for address: ffff8881178d8cc3 #PF: supervisor read access in kernel mode #PF: error_code(0x0000) - not-present page PGD 4a01067 P4D 4a01067 PUD 0 Oops: 0000 [#1] PREEMPT SMP NOPTI CPU: 3 PID: 1736 Comm: mount.cifs Not tainted 6.7.0-rc4 #1 Hardware name: QEMU Standard PC (Q35 + ICH9, 2009), BIOS rel-1.16.2-3-gd478f380-rebuilt.opensuse.org 04/01/2014 RIP: 0010:smb2_parse_contexts+0xa0/0x3a0 [cifs] Code: f8 10 75 13 48 b8 93 ad 25 50 9c b4 11 e7 49 39 06 0f 84 d2 00 00 00 8b 45 00 85 c0 74 61 41 29 c5 48 01 c5 41 83 fd 0f 76 55 <0f> b7 7d 04 0f b7 45 06 4c 8d 74 3d 00 66 83 f8 04 75 bc ba 04 00 RSP: 0018:ffffc900007939e0 EFLAGS: 00010216 RAX: ffffc90000793c78 RBX: ffff8880180cc000 RCX: fff",
         "vulnerability_id":"CVE-2023-52434",
         "name":"CVE-2023-52434",
         "package_name":"linux",
         "package_details":{
            "file_path":null,
            "name":"linux",
            "package_manager":"OS",
            "version":"5.4.0",
            "release":"189.209"
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":8.0,
         "cvss_v30_score":0.0,
         "cvss_v31_score":8.0,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://people.canonical.com/~ubuntu-security/cve/2023/CVE-2023-52434.html",
         "source":"UBUNTU_CVE",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-52434 - linux",
         "reason_to_ignore":"N/A"
      },
      {
         "description":" In the Linux kernel, the following vulnerability has been resolved: smb: client: fix use-after-free bug in cifs_debug_data_proc_show() Skip SMB sessions that are being teared down (e.g. @ses->ses_status == SES_EXITING) in cifs_debug_data_proc_show() to avoid use-after-free in @ses. This fixes the following GPF when reading from /proc/fs/cifs/DebugData while mounting and umounting [ 816.251274] general protection fault, probably for non-canonical address 0x6b6b6b6b6b6b6d81: 0000 [#1] PREEMPT SMP NOPTI ... [ 816.260138] Call Trace: [ 816.260329] <TASK> [ 816.260499] ? die_addr+0x36/0x90 [ 816.260762] ? exc_general_protection+0x1b3/0x410 [ 816.261126] ? asm_exc_general_protection+0x26/0x30 [ 816.261502] ? cifs_debug_tcon+0xbd/0x240 [cifs] [ 816.261878] ? cifs_debug_tcon+0xab/0x240 [cifs] [ 816.262249] cifs_debug_data_proc_show+0x516/0xdb0 [cifs] [ 816.262689] ? seq_read_iter+0x379/0x470 [ 816.262995] seq_read_iter+0x118/0x470 [ 816.263291] proc_reg_read_iter+0x53/0x90 [ 816.263596] ? srso_alias_return_thunk+0x5",
         "vulnerability_id":"CVE-2023-52752",
         "name":"CVE-2023-52752",
         "package_name":"linux",
         "package_details":{
            "file_path":null,
            "name":"linux",
            "package_manager":"OS",
            "version":"5.4.0",
            "release":"189.209"
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.8,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.8,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://people.canonical.com/~ubuntu-security/cve/2023/CVE-2023-52752.html",
         "source":"UBUNTU_CVE",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-52752 - linux",
         "reason_to_ignore":"N/A"
      }
   ]
}